meta-ostro/classes/buildhistory-extra.bbclass

```diff
@@ -33,6 +33,15 @@ python buildhistory_extra_emit_pkghistory() {
     if not os.path.exists(pkghistdir):
         bb.utils.mkdirhier(pkghistdir)
 
+    # Make the recorded information independent of varying paths.
+    # Some recipes use destsuffix=${S}/... to fetch components
+    # into their main source tree. The other variable do not show
+    # up (yet), but might eventually.
+    d_original = d
```

Comment:
Not at the moment. I added it as a reminder that there are now two flavors of d, the simplified one and the original.

I don't care too much about it, though.